ci/long_running_distributed_tests/workloads/cifar_pytorch_pbt.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import os
+import random
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader, Subset
+from torchvision.datasets import CIFAR10
+import torchvision.transforms as transforms
```

Comment:
BTW, can you actually run this on the Jenkins CI? just so that we don't break any abstractions?